src/snowflake/cli/api/exceptions.py

```diff
@@ -114,3 +114,17 @@ def __init__(self, path: Path):
         super().__init__(
             f'Configuration file {path} has too wide permissions, run `chmod 0600 "{path}"`'
         )
+
+
+class DatabaseNotProvidedError(ClickException):
+    def __init__(self):
+        super().__init__(
+            "Database not specified. Please update connection to add `DATABASE` parameter, or re-run command using `--database` option. Use `snow connection list` to list existing connections."
```

Comment:
```suggestion
            "Database not specified. Please update connection to add `database` parameter, or re-run command using `--database` option. Use `snow connection list` to list existing connections."
```
I think config entries have to be lowercase